Week1/insert-values.js

```diff
@@ -0,0 +1,26 @@
+var mysql      = require('mysql');
+var connection = mysql.createConnection({
+  host     : 'localhost',
+  user     : 'hyfuser',
+  password : 'hyfpassword',
+  database : 'class17'
+});
+
+connection.connect();
+var insert_queries = [
+                    "insert into students values (103, 'Ahmad', '1986-01-11', 8.3, 'm')",
+                    "insert into students values (104, 'Rim', '1979-11-01', 8.7, 'f')",
+                    "insert into students values (105, 'Rabia', '1978-05-30', 8.1, 'f')",
+                    "insert into students values (106, 'Karam', '1994-02-02', 8.8, 'm')"
```

Comment:
Yeah, Because it was the first class, we did not go into prepared statements. But the goal is have prepared statements and async, promises like you showed me yesterday.